tests/test_schema_rules.py

```diff
@@ -13,7 +13,7 @@ def test_no_mixed_type_in_enum():
 
             if "enum" in prop:
                 assert all(
-                    [type(i) == str for i in prop["enum"]]
+                    [type(i) == str or i == None for i in prop["enum"]]
```

Comment:
please add at the beginning of this test the same comment as in https://github.com/uc-cdis/data-simulator/pull/39